server/src/com/thoughtworks/go/server/controller/AgentRegistrationController.java

```diff
@@ -88,11 +88,6 @@ public void checkAgentStatus(HttpServletResponse response) throws IOException {
         setOtherHeaders(response);
     }
 
-    @RequestMapping(value = "/admin/latest-agent.status", method = RequestMethod.GET)
```

Comment:
This was doing the same thing as `checkAgentStatus` so merged the GET and HEAD calls.